### PR TITLE
Update CHANGELOG

### PR DESCRIPTION
**Pull request by RamLavi**
Update the CHANGELOG in preparation for release 0.5.0

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,11 @@
+## [0.5.0] - 2023-08-12
+### Breaking Changes
+### New Features
+- Use CentOS stream 9 on traffic-gen and VM under test images
```
Comment (orelmisan) [inline CHANGELOG:4]: We wrote the base image version in previous releases.

Comment (RamLavi) [reply CHANGELOG:4]: ah, then removing

Comment (orelmisan) [reply CHANGELOG]: I don't think this line should have been removed.
I referred to the fact we wrote the base image version, for example:https://github.com/kiagnose/kubevirt-dpdk-checkup/blob/d67d80a9664df6c822e9dfac6eabfd1ef7c882b8/CHANGELOG#L15

Comment (RamLavi) [reply CHANGELOG:4]: DONE

**Pull request merged**